docs/polygon_draw.js

```diff
@@ -366,22 +387,79 @@ function setUpPolygonDrawing(map, firebasePromise) {
  * @param {google.maps.Map} map Map to display regions on
  * @param {Promise<any>} firebasePromise Promise with Firebase damage data (also
  *     implies that authentication is complete)
- * @return {Promise} promise that is resolved when all initialization is done
- *     (only used by tests)
+ * @return {?google.maps.drawing.DrawingManager} DrawingManager added to map
```

Comment:
Do you not document the implicit promise wrap for async functions?

Reply:
Actually, seems like IntelliJ is fine either way, but then complains if you assign the result of the function to something which is supposed to be a Promise, so documented as a Promise.